### PR TITLE
Change links to have full path names in TOC.md for tizen-studio

### PR DESCRIPTION
**Pull request by journey2w**
- Delete component designer from TOC.md.
 - Remmove ing from file name.
 - Fix errors such as links, typo, and broken sentences.

Fixed Issue #213.

### CODE REVIEW

File: docs/application/tizen-studio/TOC.md [hunk 1]
```diff
@@ -1,149 +1,149 @@
...
-## Managing Projects
-### [Project Creation Wizard](native-tools/project-wizard.md)
-### [Project Explorer View](native-tools/project-explorer-view.md)
-### [Certificate Registration](common-tools/certificate-registration.md)
-### [Device Manager](common-tools/device-manager.md)
-## Writing and Editing Code
-### [Content Assist](native-tools/content-assist.md)
-### [API and Privilege Checker](native-tools/api-checker.md)
+## [Overview](/application/tizen-studio/native-tools/overview.md)
+## Manag Projects
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Manage` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 2]
```diff
@@ -1,149 +1,149 @@
...
+#### [Network Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/network-analysis.md)
+#### [OpenGL Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/opengl-analysis.md)
+#### [UI Hierarchy Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/ui-hierarchy-analysis.md)
+### [Call Stack View](/application/tizen-studio/native-tools/call-stack-view.md)
+### [Static Analyzer](/application/tizen-studio/native-tools/static-analyzer.md)
+### [Valgrind](/application/tizen-studio/native-tools/valgrind.md)
+### [T-trace](/application/tizen-studio/native-tools/t-trace.md)
+### [Address Sanitizer](/application/tizen-studio/native-tools/address-sanitizer.md)
+### [Code Coverage](/application/tizen-studio/native-tools/code-coverage.md)
+## Runn and Test Your App
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Run` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 3]
```diff
@@ -1,149 +1,149 @@
...
-### [Previewer](web-tools/previewer.md)
-### [Privilege Checker](web-tools/privilege-checker.md)
-### [Tizen Studio Web Preferences](web-tools/ide-preferences.md)
-## Configuring Your App
-### [Configuration Editor](web-tools/config-editor.md)
-### [Web Localization](web-tools/web-localization.md)
-## Debugging Your App
-### [Log View](common-tools/log-view.md)
+## [Overview](/application/tizen-studio/web-tools/overview.md)
+## Manag Projects
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Manage` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 4]
```diff
@@ -1,149 +1,149 @@
...
+### [Content Assist](/application/tizen-studio/web-tools/content-assist.md)
+### [Code Clean](/application/tizen-studio/web-tools/code-productivity.md)
+### [REST Viewer](/application/tizen-studio/web-tools/rest-viewer.md)
+### [Previewer](/application/tizen-studio/web-tools/previewer.md)
+### [Privilege Checker](/application/tizen-studio/web-tools/privilege-checker.md)
+### [Tizen Studio Web Preferences](/application/tizen-studio/web-tools/ide-preferences.md)
+## Configure Your App
+### [Configuration Editor](/application/tizen-studio/web-tools/config-editor.md)
+### [Web Localization](/application/tizen-studio/web-tools/web-localization.md)
+## Debugg Your App
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Debug` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 5]
```diff
@@ -1,149 +1,149 @@
...
+#### [Overview](/application/tizen-studio/common-tools/dynamic-analyzer/overview.md)
+#### [Common Tasks](/application/tizen-studio/common-tools/dynamic-analyzer/common-tasks.md)
+#### [Advanced Tasks](/application/tizen-studio/common-tools/dynamic-analyzer/advanced-tasks.md)
+#### [Memory Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/memory-analysis.md)
+#### [Thread Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/thread-analysis.md)
+#### [File Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/file-analysis.md)
+#### [Network Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/network-analysis.md)
+#### [OpenGL Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/opengl-analysis.md)
+#### [UI Hierarchy Analysis](/application/tizen-studio/common-tools/dynamic-analyzer/ui-hierarchy-analysis.md)
+## Runn and Test Your App
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Run` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 6]
```diff
@@ -1,149 +1,149 @@
...
-### [Using the Web Simulator](web-tools/web-simulator.md)
-### [Web Simulator Features](web-tools/web-simulator-features.md)
-## [CLI](common-tools/command-line-interface.md)
-## [Keyboard Shortcuts](common-tools/keyboard-shortcuts.md)
+#### [Overview](/application/tizen-studio/common-tools/emulator.md)
+#### [Emulator Manager](/application/tizen-studio/common-tools/emulator-manager.md)
+#### [Emulator Control Panel](/application/tizen-studio/common-tools/emulator-control-panel.md)
+#### [Extended Emulator Features](/application/tizen-studio/common-tools/emulator-features.md)
+### [Web Unit Test Tool](/application/tizen-studio/web-tools/web-unit-test-tool.md)
+### [Us the Web Simulator](/application/tizen-studio/web-tools/web-simulator.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Use` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 7]
```diff
@@ -1,149 +1,149 @@
...
-### [Using the Serial Terminal](rt-ide/getting-started/terminal.md)
-### [Debugging the Project](rt-ide/getting-started/debug.md)
-## SmartThings&trade; Extension
-### [Overview](rt-ide/smartthings-ext/overview.md)
-### [Creating a SmartThings&trade; Project](rt-ide/smartthings-ext/create-smartthings.md)
-### [Building the SmartThings&trade; Project](rt-ide/smartthings-ext/build-smartthings.md)
-### [Managing SmartThings&trade; Certificates](rt-ide/smartthings-ext/certificate-smartthings.md)
-### [Device/Resource Model Manager](rt-ide/smartthings-ext/model-manager.md)
+## [Overview](/application/tizen-studio/rt-ide/overview.md)
+## Gett Started
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Get` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 8]
```diff
@@ -1,149 +1,149 @@
...
+### [Overview](/application/tizen-studio/rt-ide/gett-started/overview.md)
+### [Install Tizen Studio for RT](/application/tizen-studio/rt-ide/gett-started/install.md)
+### [Manag a Project](/application/tizen-studio/rt-ide/gett-started/create.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Manage` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 9]
```diff
@@ -1,149 +1,149 @@
...
+### [Flash the Project](/application/tizen-studio/rt-ide/gett-started/flash.md)
+### [Us the Serial Terminal](/application/tizen-studio/rt-ide/gett-started/terminal.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Use` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 10]
```diff
@@ -1,149 +1,149 @@
...
+### [Debugg the Project](/application/tizen-studio/rt-ide/gett-started/debug.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Debug` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 11]
```diff
@@ -1,149 +1,149 @@
...
+## SmartThs&trade; Extension
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `SmartThings` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 12]
```diff
@@ -1,149 +1,149 @@
...
+### [Overview](/application/tizen-studio/rt-ide/smartths-ext/overview.md)
+### [Creat a SmartThs&trade; Project](/application/tizen-studio/rt-ide/smartths-ext/create-smartths.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Create` and `SmartThings` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 13]
```diff
@@ -1,149 +1,149 @@
...
+### [Build the SmartThs&trade; Project](/application/tizen-studio/rt-ide/smartths-ext/build-smartths.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `SmartThings` is correct.

File: docs/application/tizen-studio/TOC.md [hunk 14]
```diff
@@ -1,149 +1,149 @@
...
+### [Manag SmartThs&trade; Certificates](/application/tizen-studio/rt-ide/smartths-ext/certificate-smartths.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/TOC.md]: It looks like `Manage` and `SmartThings` are correct.

File: docs/application/tizen-studio/common-tools/device-manager.md [hunk 15]
```diff
@@ -171,7 +171,8 @@ You can perform a variety of actions in the **Log** view:
 
     To export logs to a file, click the **Export Logs** button (![Export Logs](./media/device_manager_export_logs_icon.png)), select the file storage location and file name for the exported log, and click **OK**.
 
-    *Note** If you have filters applied to a log tab, only messages that match those filters are exported. If you want to export the complete log, remove all filters before exporting.
+   > *Note**
```
Comment (mijin85cho) [inline docs/application/tizen-studio/common-tools/device-manager.md]: It should be `**Note**`

File: docs/application/tizen-studio/common-tools/emulator-features.md [hunk 16]
```diff
@@ -120,7 +124,7 @@ The emulator is based on the QEMU virtual machine. The NAT type exploits the QEM
 
 The emulator provides 3 methods of proxy configuration. They can be configured in **Emulator Manager > Network Configuration**:
 
-- **No proxy**: does not use the proxy configuration.
+- **No proxy**: does not use the proxy configuration/.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/common-tools/emulator-features.md]: Useless `/` at the end of the line.

File: docs/application/tizen-studio/index.md [hunk 17]
```diff
@@ -9,7 +9,7 @@ In the Tizen Studio, you can perform all the same development activities that we
 
 To develop your applications, use the set of tools provided by the Tizen Studio:
 
-- [Setup](setup/installing-sdk.md)  
+- [Setup](setup/install-studio.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/index.md]: It looks like `install-sdk.md` is correct.

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md [hunk 18]
```diff
@@ -196,8 +196,8 @@ For more information on the relationship between the elements, see the [element
 
 **Attributes:**
 
-- `xml:lang`  
-Language and country code (available value: "\<2-letter lowercase language code (ISO 639-1)\>-\<2-letter lowercase country code (ISO 3166-1 alpha-2)\>")
+- `xml:lang`
+  Language and country code (available value: "\<2-letter lowercase language code (ISO 639-1)\>-\<2-letter lowercase country code (ISO 3166-1 alpha-2)\>")
```
Comment (mijin85cho) [inline docs/application/tizen-studio/native-tools/manifest-text-editor.md]: One empty line is needed here.

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md [hunk 19]
```diff
@@ -311,47 +323,56 @@ For more information on the relationship between the elements, see the [element
...
+  Indicates whether the application is relaunched automatically if it is terminated abnormally (available values: `true`, `false`)
+
+  If the value is not defined, `false` is used.
 
  > **Note**  
  > This attribute is not supported on Tizen wearable devices. Since Tizen 2.4, this attribute is not supported on all Tizen devices.
 
-- `exec`  
-Application executable file path (string)
+- `exec`
```
Comment (mijin85cho) [inline docs/application/tizen-studio/native-tools/manifest-text-editor.md]: One empty line is needed here.

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md [hunk 20]
```diff
@@ -311,47 +323,56 @@ For more information on the relationship between the elements, see the [element
...
+
+- `taskmanage`
 
-- `taskmanage`  
-Indicates whether the application is shown in the task manager (available values: `true`, `false`)
+  Indicates whether the application is shown in the task manager (available values: `true`, `false`)
 
-- `type`  
-Tizen application type (available values: `capp`, `c++app`, `webapp`)
+- `type`
```
Comment (mijin85cho) [inline docs/application/tizen-studio/native-tools/manifest-text-editor.md]: Question : One empty space is neede here. Is it intended to form it in one line?

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md [hunk 21]
```diff
@@ -680,21 +725,25 @@ This element contains specific service provider or user account protocol-related
...
-- `multiple-accounts-support`  
-Indicates whether multiple accounts are supported (available values: `true`, `false`)
+- `multiple-accounts-support`
 
-- `providerid`  
-ID of the account provider (string)
+  Indicates whether multiple accounts are supported (available values: `true`, `false`)
+
+- `providerid`
+  ID of the account provider (string)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/native-tools/manifest-text-editor.md]: One empty line is needed here.

File: docs/application/tizen-studio/rt-ide/getting-started/install.md [hunk 22]
```diff
@@ -26,7 +26,7 @@ To install the Tizen Studio for RT:
 
    If you want to install additional platforms and tools, launch the Package Manager by checking the **Launch the Package Manager** check box before clicking **Finish**.
 
-   Through the Tizen Studio Package Manager, you can both install and update additional tools. For more information on the Package Manager, see [Updating Tizen Studio](../setup/updating-sdk.md) and [Configuring the Package Manager](../setup/advanced-configuration.md).
+   Through the Tizen Studio Package Manager, you can both install and update additional tools. For more information on the Package Manager, see [Updating Tizen Studio](../setup/update-sdk.md) and [Configuring the Package Manager](../setup/advanced-configuration.md).
```
Comment (mijin85cho) [inline docs/application/tizen-studio/rt-ide/getting-started/install.md]: Broken links : It should be `../../setup/update-sdk.md` and `../../setup/advanced-configuration.md`.

File: docs/application/tizen-studio/TOC.md (hunk 1 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 2 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 3 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 4 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 5 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 7 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: Getting started is correct. We will change a gerund to a verb next time.

File: docs/application/tizen-studio/TOC.md (hunk 8 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 9 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 10 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 11 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done.

File: docs/application/tizen-studio/TOC.md (hunk 12 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 13 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/TOC.md (hunk 14 above)
Comment (journey2w) [reply docs/application/tizen-studio/TOC.md]: done

File: docs/application/tizen-studio/common-tools/device-manager.md (hunk 15 above)
Comment (journey2w) [reply docs/application/tizen-studio/common-tools/device-manager.md]: done

File: docs/application/tizen-studio/common-tools/emulator-features.md (hunk 16 above)
Comment (journey2w) [reply docs/application/tizen-studio/common-tools/emulator-features.md]: done.

File: docs/application/tizen-studio/index.md (hunk 17 above)
Comment (journey2w) [reply docs/application/tizen-studio/index.md]: done

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md (hunk 18 above)
Comment (journey2w) [reply docs/application/tizen-studio/native-tools/manifest-text-editor.md]: done

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md (hunk 19 above)
Comment (journey2w) [reply docs/application/tizen-studio/native-tools/manifest-text-editor.md]: done

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md (hunk 20 above)
Comment (journey2w) [reply docs/application/tizen-studio/native-tools/manifest-text-editor.md]: done

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md (hunk 21 above)
Comment (journey2w) [reply docs/application/tizen-studio/native-tools/manifest-text-editor.md]: done

File: docs/application/tizen-studio/rt-ide/getting-started/install.md (hunk 22 above)
Comment (journey2w) [reply docs/application/tizen-studio/rt-ide/getting-started/install.md]: done

**Pull request merged**